docs/source/developing/writing-docs.adoc

```diff
@@ -110,4 +183,7 @@ $ cd ..
 $ rake build
 ----
 
-If the build completes successfully, the site is available under _preview/openshift-origin/latest/welcome/index.html_.
+If the build completes successfully, the site is available under *_preview/openshift-origin/latest/welcome/index.html_*.
+
+OpenShift runs a nightly build in which they update the documentation. After you update the documentation,
```

Comment:
This information was already in the topic, and the topic itself is titled "writing and publishing docs" so it can include both topics, unless you have a new suggestion on how to organize the content?